test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/TxnVerbs.java

```diff
@@ -330,6 +345,66 @@ public static HapiContractUpdate contractUpdate(String contract) {
 		return new HapiContractUpdate(contract);
 	}
 
+	/*  Note to the reviewer:
+		This method is temporarily named with the "new" prefix, as soon as the implementation is approved and the legacy
+		fileCreate() is entirely replaced, this method will be renamed to "fileCreate"
+	 */
+	public static HapiSpecOperation newFileCreate(final String... contractsNames) {
```

Comment:
Many tests will still create files that aren't contract initcode; so we should keep the `fileCreate()` factory and name this e.g. `uploadInitcode()`.

Reply:
Solved